tests/test_samples.rs

```diff
@@ -165,6 +176,14 @@ fn all_the_fields() {
         meta.workspace_root.file_name().unwrap(),
         PathBuf::from("all")
     );
+    assert_eq!(
+        serde_json::from_value::<WorkspaceMetadata>(meta.workspace_metadata).unwrap(),
```

Comment:
this test only works on stable, but not on beta or nightly.

Reply:
Had a bug where I was picking the wrong metadata object. Replaced with workspace.metadata path